vars/network_team_only_vars.yml

```diff
@@ -0,0 +1,29 @@
+# Copyright (C) 2021, RTE (http://www.rte-france.com)
```

Comment:
Is it not possible to have : 
- One systemd-network var file for the team0 bridge
- One systemd-network var file for the rest of the network
The first file would always be included, and the second one only if netplan is not used.

In general, I think it would be better to have multiple systemd_networkd files (admin, cluster, ptp) as it is done for netplan.

Reply:
It is impossible to do it in this way.

I have changed to have 3 files.
* One for team0 always included
* One for a network with only team0 and custom network, it relies on the variable defined in the team0 file.
* One for the default network (br0 and others). It also uses the team0 variable defined in the team0 file.